docs/content/_index.md

```diff
@@ -1,608 +1,1007 @@
 
-```json
+For example:
+
+```live:example/final:module:openable,merge_down
+package example
+
+allow = true {                                      # allow is true if...
+    count(violation) == 0                           # there are zero violations.
+}
+
+violation[server.id] {                              # a server is in the violation set if...
+    some server
+    public_server[server]                           # it exists in the 'public_server' set and...
+    server.protocols[_] == "http"                   # it contains the insecure "http" protocol.
+}
+
+violation[server.id] {                              # a server is in the violation set if...
+    server := input.servers[_]                      # it exists in the input.servers collection and...
+    server.protocols[_] == "telnet"                 # it contains the "telnet" protocol.
+}
+
+public_server[server] {                             # a server exists in the public_server set if...
+    some i, j
+    server := input.servers[_]                      # it exists in the input.servers collection and...
+    server.ports[_] == input.ports[i].id            # it references a port in the input.ports collection and...
+    input.ports[i].network == input.networks[j].id  # the port references a network in the input.networks collection and...
+    input.networks[j].public                        # the network is public.
+}
+```
+```live:example/final:query:merge_down
+some x; violation[x]
+```
+```live:example/final:output
+```
+
+## Running OPA
+
+This section explains how you can query OPA directly and interact with it on
+your own machine.
+
+### 1. Download OPA
+
+To get started download an OPA binary for your platform from GitHub releases:
+
+On macOS (64-bit):
+
+```shell
+curl -L -o opa https://github.com/open-policy-agent/opa/releases/download/{{< current_version >}}/opa_darwin_amd64
+```
+
+On Linux (64-bit):
+
+```shell
+curl -L -o opa https://github.com/open-policy-agent/opa/releases/download/{{< current_version >}}/opa_linux_amd64
+```
+
+> Windows users can obtain the OPA executable from [GitHub
+> Releases](https://github.com/open-policy-agent/opa/releases). The steps below
+> are the same for Windows users except the executable name will be different.
+
+Set permissions on the OPA executable:
+
+```shell
+chmod 755 ./opa
+```
+
+> You can also download and run OPA via Docker. The latest stable image tag is
+> `openpolicyagent/opa:{{< current_docker_version >}}`.
+
+### 2. Try `opa eval`
+
+The simplest way to interact with OPA is via the command-line using the `opa
+eval` sub-command. `opa eval` is a swiss-army knife that you can use to evaluate
+arbitrary Rego expressions and policies. `opa eval` supports a large number of
+options for controlling evaluation. Commonly used flags include:
+
+| Flag | Short | Description |
+| --- | --- | --- |
+| `--data` | `-d` | Load policy or data files into OPA. This flag can be repeated. |
```

Comment:
Don't forget `--bundle`! 😃 

More brainstorming, we might want to consider a more automated way to go from CLI doc -> website docs. Or maybe its just a matter of copying once and ensuring new changes are made in both places 🤔

Reply:
Done. Re: automating the CLI doc to website doc, I think that could be useful on another page. I don't know if we want the entire command line reference here. I tried to include the options that I use all the time.

Reply:
Agreed, maybe we could generate a like CLI reference page with the full list of options.